cibuildwheel/resources/pinned_docker_images.cfg

```diff
@@ -1,43 +1,43 @@
+musllinux_1_1 = quay.io/pypa/musllinux_1_1_x86_64:2022-06-13-c365205
 
 [i686]
-manylinux1 = quay.io/pypa/manylinux1_i686:2022-06-05-89b8e6d
-manylinux2010 = quay.io/pypa/manylinux2010_i686:2022-06-05-61145a4
-manylinux2014 = quay.io/pypa/manylinux2014_i686:2022-06-05-61145a4
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_i686:2022-06-05-61145a4
-musllinux_1_1 = quay.io/pypa/musllinux_1_1_i686:2022-06-05-61145a4
+manylinux1 = quay.io/pypa/manylinux1_i686:2022-06-12-a846b05
+manylinux2010 = quay.io/pypa/manylinux2010_i686:2022-06-13-c365205
+manylinux2014 = quay.io/pypa/manylinux2014_i686:2022-06-13-c365205
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_i686:2022-06-13-c365205
+musllinux_1_1 = quay.io/pypa/musllinux_1_1_i686:2022-06-13-c365205
 
 [pypy_x86_64]
-manylinux2010 = quay.io/pypa/manylinux2010_x86_64:2022-06-05-61145a4
-manylinux2014 = quay.io/pypa/manylinux2014_x86_64:2022-06-05-61145a4
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_x86_64:2022-06-05-61145a4
+manylinux2010 = quay.io/pypa/manylinux2010_x86_64:2022-06-13-c365205
+manylinux2014 = quay.io/pypa/manylinux2014_x86_64:2022-06-13-c365205
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_x86_64:2022-06-13-c365205
 
 [pypy_i686]
-manylinux2010 = quay.io/pypa/manylinux2010_i686:2022-06-05-61145a4
-manylinux2014 = quay.io/pypa/manylinux2014_i686:2022-06-05-61145a4
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_i686:2022-06-05-61145a4
+manylinux2010 = quay.io/pypa/manylinux2010_i686:2022-06-13-c365205
+manylinux2014 = quay.io/pypa/manylinux2014_i686:2022-06-13-c365205
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_i686:2022-06-13-c365205
 
 [aarch64]
-manylinux2014 = quay.io/pypa/manylinux2014_aarch64:2022-06-05-61145a4
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_aarch64:2022-06-05-61145a4
-musllinux_1_1 = quay.io/pypa/musllinux_1_1_aarch64:2022-06-05-61145a4
+manylinux2014 = quay.io/pypa/manylinux2014_aarch64:2022-06-13-c365205
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_aarch64:2022-06-13-c365205
+musllinux_1_1 = quay.io/pypa/musllinux_1_1_aarch64:2022-06-13-c365205
 
 [ppc64le]
-manylinux2014 = quay.io/pypa/manylinux2014_ppc64le:2022-06-05-61145a4
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_ppc64le:2022-06-05-61145a4
-musllinux_1_1 = quay.io/pypa/musllinux_1_1_ppc64le:2022-06-05-61145a4
+manylinux2014 = quay.io/pypa/manylinux2014_ppc64le:2022-06-13-c365205
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_ppc64le:2022-06-13-c365205
+musllinux_1_1 = quay.io/pypa/musllinux_1_1_ppc64le:2022-06-13-c365205
 
 [s390x]
-manylinux2014 = quay.io/pypa/manylinux2014_s390x:2022-06-05-61145a4
-manylinux_2_24 = quay.io/pypa/manylinux_2_24_s390x:2022-06-05-61145a4
-musllinux_1_1 = quay.io/pypa/musllinux_1_1_s390x:2022-06-05-61145a4
+manylinux2014 = quay.io/pypa/manylinux2014_s390x:2022-06-08-6ab6a6f
+manylinux_2_24 = quay.io/pypa/manylinux_2_24_s390x:2022-06-08-6ab6a6f
+musllinux_1_1 = quay.io/pypa/musllinux_1_1_s390x:2022-06-08-6ab6a6f
```

Comment:
There was an issue with s390x on TravisCI on sunday/monday.
Probably best to restart the workflow or wait for next week's update in order to get consistent images.

Reply:
I've rerun, as getting 3.10.5 might be nice, if we are planning on a patch release out soon with #1137.